internal/v2/repository/user/mfa.go

```diff
@@ -0,0 +1,9 @@
+package user
+
+type MFAState int32
+
+const (
+	MFAStateUnspecified MFAState = iota
```

Comment:
I'm not sure if we need the unspecified-value from the api in the business logic